examples/scripts/7_Others/7.6_Compare_ASE_to_VV_FIRE.py

```diff
@@ -113,151 +120,635 @@
 
 
 def run_optimization(
```

Comment:
Would maybe make two different functions `run_optimization_ts` and `run_optimization_ase` as this is already two separate functions with a toggle?